### ISSUE TITLE
Refreshing Preferences (user settings) page loses Account and credit card info

### ISSUE THREAD

**Issue by tfiduccia** [rancher/rancher#8422]
**Rancher Versions:** 4/5

**Steps to Reproduce:**
1.  Go to user settings 
2. Refresh page

**Results:** The Account and credit card sections disappear.  Error in console:
```
Assertion Failed: You modified "access.identity" twice on <ui@service:access::ember423> in a single render. It was rendered in "component:identity-avatar" and modified in "component:settings/user-info". This was unreliable and slow in Ember 1.x and is no longer supported. See https://github.com/emberjs/ember.js/issues/13948 for more details.
Error
```
![image](https://cloud.githubusercontent.com/assets/11514927/24723955/43deec4a-19fe-11e7-98c9-8f17d91dc489.png)

**Comment by westlywright** [comment]
@tfiduccia were you seeing this before or after setting up a payment option?

**Comment by tfiduccia** [comment]
@westlywright - I didn't setup any payment info.  And I noticed that it did eventually come back, but it just took forever.  Also I got an console error:
ember-cli-live-reload.js:7 GET http://caas.rancher.io:49153/livereload.js net::ERR_CONNECTION_REFUSED

Then after a couple of minutes it shows up with this error in console:
ember.debug.js:18015 Assertion Failed: You modified "hasCattleSystem" twice on <ui@controller:authenticated::ember773> in a single render. It was rendered in "component:page-header" and modified in "component:settings/user-info". This was unreliable and slow in Ember 1.x and is no longer supported. See https://github.com/emberjs/ember.js/issues/13948 for more details.
